pkg/controller.v1/common/job_controller.go

```diff
@@ -69,9 +69,9 @@ var (
 type GangScheduler string
 
 const (
-	GangSchedulerNone             GangScheduler = "None"
 	GangSchedulerVolcano          GangScheduler = "volcano"
 	GangSchedulerSchedulerPlugins GangScheduler = "scheduler-plugins"
+	GangSchedulerDefault          GangScheduler = "default-scheduler"
```

Comment:
Add a comment that using this scheduler name or any scheduler name different than volcano uses the scheduler-plugins PodGroup

Reply:
Done